src/components/helpers/filterHelper.js

```diff
@@ -3,32 +3,28 @@ import { stringify as queryStringStringify } from "query-string";
 const filterHelper = (() => {
   return {
     isObject(object) {
-      return object !== null && typeof object === "object";
+      return !object && typeof object === "object";
```

Comment:
is it correctly condition?

Reply:
I understood my not correct condition. Thanks.